conversational-skills/procode-spec.yaml

```diff
@@ -1164,8 +1164,8 @@ components:
               type: string
               description: An error message to display to the user in case the slot value is not valid per the business rules.
             prompt:
-              type: object
```

Comment:
Oh, when did we change this from `object` to string? Is this a recent change or something old we missed 🙈

Reply:
It's not actually something we've changed yet but WxO doesn't use this so we decided to simplify this

Reply:
Sounds good- let's merge this in after the backend code has updated then